R/tbl_cross.R

```diff
@@ -0,0 +1,169 @@
+#' Create a cross table of summary statistics
```

Comment:
Overall this looks great! It is much cleaner now that the p-value options are outside the function. 

One think I keep expecting to be able to do as a user is:

`trial %>% select(response, trt) %>%
  tbl_cross(col = trt) `

Do we think we should allow this?

Reply:
i think it makes the most sense to specify both or none.